client/model.go

```diff
@@ -880,3 +880,14 @@ func (e *AuditLogEntry) AccessLog() (*auditv1.AccessLogEntry, error) {
 func (e *AuditLogEntry) DecisionLog() (*auditv1.DecisionLogEntry, error) {
 	return e.decisionLog, e.err
 }
+
+type PolicyKind uint8
+
+const (
+	ResourcePolicyKind PolicyKind = iota + 1
+	PrincipalPolicyKind
+	DerivedRolesPolicyKind
+)
+
+// FilterOpt is used to specify search terms for ListPolicies method.
+type FilterOpt func()
```

Comment:
This is dead code now, isn't it?

Reply:
Yeah, it is actually. Let's remove the `PolicyKind` and `FilterOpt`.  I left the `FilterOpt` as it may be used in the future but if we are going to use it with `ListPolicies(ctx context.Context, opts ...FilterOpt)` fashion, actually it won't break if we add options in the future.